membership-attribute-service/app/actions/CommonActions.scala

```diff
@@ -54,17 +54,13 @@ class AuthenticatedUserAndBackendRequest[A](
     val touchpoint: TouchpointComponents,
     val request: Request[A],
 ) extends WrappedRequest[A](request) {
-  implicit val logPrefix: LogPrefix = new LogPrefix {
-    override def message: String = user.identityId
-  }
+  implicit val logPrefix: LogPrefix = user.logPrefix
 }
 
 class AuthAndBackendRequest[A](
     val redirectAdvice: RedirectAdviceResponse,
     val touchpoint: TouchpointComponents,
     request: Request[A],
 ) extends WrappedRequest[A](request) {
-  implicit val logPrefix: LogPrefix = new LogPrefix {
-    override def message: String = redirectAdvice.userId.getOrElse("no-identity-id")
-  }
```

Comment:
I previously made the LogPrefix a trait but that was totally unnecessary because it's just returning a fixed value, so now it's a case class.